Felienne/hedy
# [FEATURE] Enables users to toggle password visibility on signup

**Description**
In this PR we make a nice QoL change where we add a feature to enable users to toggle the visibility of the password when signing up. As we don't have a password repeat anymore this is a nice addition to enable users to verify that their entered password is what they want it to be.
![Schermafbeelding 2022-07-21 om 14 15 51](https://user-images.githubusercontent.com/38155424/180211587-61bf548d-41ce-4042-8f4c-d4f52214cbb9.png)
![Schermafbeelding 2022-07-21 om 14 16 01](https://user-images.githubusercontent.com/38155424/180211593-08c9da44-598d-475f-8f77-220af07cbea8.png)


**Fixes**
This PR fixes #3082.

**How to test**
Navigate to the signup page. Notice the new icon on the password input field and verify that pressing it toggles the symbol itself as well as the type of the field, either showing or hiding the password.
